mmv1/third_party/terraform/website/docs/guides/external_credentials_stacks.html.markdown

```diff
@@ -0,0 +1,200 @@
+---
+page_title: "Use external credentials in the Google Cloud provider with Terraform Stacks"
+description: |-
+  How to use external credentials in the Google Cloud provider with Terraform Stacks
+---
+
+# External Credentials in the Google Cloud provider with Terraform Stacks
+
+Apart from using `access_token` and `credential` fields in the provider configuration, you can also use external credentials in the Google Cloud provider that are provided through a Workload Identity Federation (WIF) provider. This can be used to authenticate Terraform Stacks to provision resources in Google Cloud.
```

Comment:
We should also edit https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#authentication-configuration to briefly talk about External Credentials and link to this page.

Can we also specify that `External Credentials` will take precedence over any other authentication method (since it's checked first in https://github.com/GoogleCloudPlatform/magic-modules/pull/12714/files#diff-2f2e11cd40a91d9e9b280c00ad695d0d71e8d43488a3026e226fc98a79b20776R288